### DISCUSSION TITLE
Update text color based on system color scheme

### DISCUSSION THREAD

**Discussion by feifanzhou** [krzyzanowskim/STTextView#40] [Questions]
Hello! In the `STTextViewUI.TextView` example, the text color doesn't change when the system color scheme changes, leading to a default of dark text in dark mode:

<img width="641" alt="Screenshot 2023-10-16 at 18 03 14" src="https://github.com/krzyzanowskim/STTextView/assets/3407996/6780c65c-0ee8-4d40-b55c-12905fea52b0">

I [attempted a naive fix](https://github.com/krzyzanowskim/STTextView/compare/main...feifanzhou:STTextView:fz/swiftui_foreground?expand=1), but it has no effect on the preview. I'm happy to investigate further if you have some pointers about where to look.

It looks like `colorScheme` is [referenced in `TextView`](https://github.com/krzyzanowskim/STTextView/blob/b9d002f4e50404df56395c5847c02d084e471e75/Sources/STTextViewUI/TextView.swift#L26), but isn't used anywhere. Was this meant to be used somewhere?

**Comment by krzyzanowskim** [comment]
It looks like this is the Demo app issue. The problem is that `AttributedString` by default doesn't have any text color set. If we explicitly set the `NSColor.textColor` to it, it going to update as expected:

```swift
AttributedString(string, attributes: AttributeContainer().foregroundColor(NSColor.textColor))
```

I'm going to update demo app code with this

**Reply by feifanzhou** [reply]
Ah yes, [that change](https://github.com/krzyzanowskim/STTextView/commit/19692395d75db80b29d977f261fee6d29029dcf0) did the trick — thanks!
